tests/test_cli.py

```diff
@@ -876,8 +876,8 @@ def test_setup_logging_without_debug(self):
         "output_format,no_colour,expected_output", [
             ("json", True, '{\n    "stack": "CREATE_COMPLETE"\n}'),
             ("json", False, '{\n    "stack": "\x1b[32mCREATE_COMPLETE\x1b[0m\"\n}'),
-            ("yaml", True, {'stack': 'CREATE_COMPLETE'}),
-            ("yaml", False, '{\'stack\': \'\x1b[32mCREATE_COMPLETE\x1b[0m\'}')
```

Comment:
Geez, the fact that someone doctored the test to show non-yaml for the "yaml" parameter is just gross. Talk about doctoring tests just so they pass... Thanks for fixing this, man.